.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 node_js:
   - 'node'
+env:
+  - EXTENSION_ID=hlepfoohegkhhmjieoechaddaejaokhf
```

Comment:
Just curious, why is this an environment variable and not just a CLI flag? It's not something that would change depending on environment, so would make more sense to me to have as a flag.